### ISSUE TITLE
problem in conjunction with the plugin  amiorin/vim-project :Welcome command

### ISSUE THREAD

**Issue by mortbauer** [fholgado/minibufexpl.vim#84]
Hey it is a quite specific bug i found, havn't really lokked into it more, but it occurs in your (by the way great plugin) and looks the following:

`Error detected while processing function <SNR>38_AutoUpdate..<SNR>38_CycleBuffer:                              
line   13:                                                                                                     
E85: There is no listed buffer`

it only happens if I use the ":Welcome" function of https://github.com/amiorin/vim-project.
Maybe somebody can help.

Thanks

**Comment by weynhamz** [comment]
Which branch do you use, the current master is really buggy, please give the develop branch a shot.

**Comment by NonLogicalDev** [comment]
Actually I second that. For some reason it does not play well with the `Startify` plugin either.

It seems like MiniBufferExplorer confuses vim into thinking that MiniBufferExplorer is the active pane when in fact it is not. All sorts of crazy things happen after that.

One of the side effects by the way is that it does not get themed.(Lacks Powerline too unless selected)

**Comment by weynhamz** [comment]
@AkaiBureido Plese test the develop branch for this

**Comment by NonLogicalDev** [comment]
I am on develop branch. At least I think I am...
[Edit: Apologies, I an not very good with git apparently. Now I am on develop. I will check whether the issue is still there.]

**Comment by weynhamz** [comment]
@AkaiBureido Any update? Does the develop branch fix you problem?

**Comment by mortbauer** [comment]
sorrry, after the problems i disabled it again. I will give a try again, but right now I have to many things to do.

**Comment by NonLogicalDev** [comment]
@techlivezheng Unfortunately no it does not seem like the problems got fixed... Dare I say they are even more pronounced. After opening a file with Startify and then opening NERDTree things go quite nuts... Something is profoundly wrong either with your code or with my set up...

(I believe that I should reference #87 as it seems related to my problem...)

I can share my vim folder if you want...

![screen shot 2013-06-12 at 12 46 57 pm](https://f.cloud.github.com/assets/1669886/641660/afbd52ba-d33c-11e2-8186-aca96195b209.png)

**Comment by weynhamz** [comment]
@AkaiBureido Yes, please share you vim folder and the instruction on how to get this problem.

**Comment by weynhamz** [comment]
@mortbauer Just tried with only project and minibufexpl(develop branch) installed, the "Welcome" command seems work quite well.

**Comment by weynhamz** [comment]
@mortbauer @AkaiBureido Any update? If no reponse given, I will colse this issue.

**Comment by NonLogicalDev** [comment]
here is my vimrc [http://cl.ly/3w2F3T061P0P]
It probably has a bit too many plugins... But you can test you plugin against it.

To get to the problem just open a few files to fill up the recent menu. The use the startify and you should be (hopefully) able to reproduce the problem.

**Comment by NonLogicalDev** [comment]
I will test it after the weekends with the new version if you wont beat me to that.
